app/src/lib/pocket.ts

```diff
@@ -79,7 +79,19 @@ export async function getAppsInNetwork(): Promise<Application[]> {
       break
     }
     response?.applications.forEach((app) => {
-      applicationsList.push(app)
+      const { address, chains, public_key: publicKey, jailed, max_relays: maxRelays, status,
+        staked_tokens: stakedTokens, unstaking_time: unstakingCompletionTime } = app.toJSON()
+
+      applicationsList.push({
+        address,
+        chains,
+        publicKey,
+        jailed,
+        maxRelays: BigInt(maxRelays),
```

Comment:
Ah, I think we might have missed this? Do we wanna keep things as a `BigInt` here or just convert it to number?